ils_middleware/tasks/alma/post.py

```diff
@@ -60,6 +60,7 @@ def NewMARCtoAlma(**kwargs):
     # push the mms_id from the Alma API result to the next task, Sinopia Metadata Update
     result = alma_result.content
     xml_response = ET.fromstring(result)
-    mms_id = xml_response.xpath("//mms_id/text()")
+    mms_id_list = xml_response.xpath("//mms_id/text()")
+    mms_id = " ".join(mms_id_list)
     logger.debug(f"mms_id: {mms_id}")
-    task_instance.xcom_push(key="mms_id", value=mms_id)
+    task_instance.xcom_push(key=instance_uri, value=(mms_id))
```

Comment:
Should `value=mms_id`? Otherwise `value=(mms_id)` sets a tuple of with mms_id as the only value

Reply:
yes,  `value=mms_id` would be better. I'll update here momentarily.